--- plugins/calendar/drivers/kolab/kolab_calendar.php.orig	2024-07-03 00:27:20.495519599 +0200
+++ plugins/calendar/drivers/kolab/kolab_calendar.php	2024-07-03 00:30:27.758806135 +0200
@@ -481,6 +481,15 @@
         $links = !empty($event['links']) ? $event['links'] : [];
         unset($event['links']);
 
+	// fix timezone Etc/GMT-1: Not a valid olson timezone
+	if (array_key_exists('start', $event) && $event['start']->timezone == 'Etc/GMT-1') {
+           $event['start']->setTimezone(new DateTimeZone('Europe/Stockholm'));
+        }
+        if (array_key_exists('end', $event) && $event['end']->timezone == 'Etc/GMT-1') {
+            $event['end']->setTimezone(new DateTimeZone('Europe/Stockholm'));
+        }
+
+
         //generate new event from RC input
         $object = $this->_from_driver_event($event);
         $saved  = $this->storage->save($object, 'event');
